.github/workflows/bench.yml

```diff
@@ -46,8 +46,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ['8.10.7', '9.2.5']
-        os: [ubuntu-latest]
+        ghc:
+          - '8.10.7'
+          - '9.2.5'
```

Comment:
I'm unsure what to do here. 
- Benchmark on all GHC versions, why not?
- Benchmark on GHC versions determined by some rule, e.g. the latest two versions?
- Benchmark on some specific named versions

cc @pepeiborra

Reply:
We don't do automated regression analysis on benchmark results, so it doesn't really matter much. Benchmark on each ghc major version latest minor would be reasonable

Reply:
Okay, I'll add a note but I don't want to get into JSON processing _right_ now :D